src/Kestrel.Certificates/McMaster.AspNetCore.Kestrel.Certificates.csproj

```diff
@@ -17,5 +17,9 @@ This library includes API for dynamically selecting which HTTPS certificate to u
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="8.0.0" />
```

Comment:
Can we adjust the version in this file? https://github.com/natemcmaster/LettuceEncrypt/blob/ba32cda0372ed6327126e2d1520d9c4c96d357e2/src/Directory.Build.targets#L4

If you update it there, we don't need to add this line to each .csproj file